subcommand/injector/command.go

```diff
@@ -117,6 +118,7 @@ func (c *Command) Run(args []string) int {
 		c.UI.Error(fmt.Sprintf("Error loading in-cluster K8S config: %s", err))
 		return 1
 	}
+	config.UserAgent = fmt.Sprintf("vault-k8s/%s", version.Version)
```

Comment:
Should this include the "v" prefix?

```suggestion
	config.UserAgent = fmt.Sprintf("vault-k8s/v%s", version.Version)
```

Reply:
Nope. It's not obvious, but the `v` is added by the release process here: https://github.com/hashicorp/vault-k8s/blob/main/Makefile#L14

Reply:
Ah! Gotcha, thanks.